gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinServerAuthIntegrateTest.java

```diff
@@ -142,7 +142,7 @@ public void shouldFailAuthenticateWithPlainTextBadPassword() throws Exception {
         } catch(Exception ex) {
             final Throwable root = ExceptionUtils.getRootCause(ex);
             assertEquals(ResponseException.class, root.getClass());
-            assertEquals("Username and/or password are incorrect", root.getMessage());
+            assertEquals("Failed to authenticate", root.getMessage());
```

Comment:
i think the new error message is better, but what caused it to change?

Reply:
The 'Username and/or password are incorrect' exception was thrown at validation time, and was caught and ignored. Subsequent requests will not trigger authentication again because the negotiator is already created.